### PR TITLE
Enchance EDC UI terminologies

### PR DESCRIPTION
**Pull request by illfixit**
This reverts commit 1ccf89e6b280968eb70087d7e4a222f28a3145d9.

_What issues does this PR close?_
closes #1421 Enhance EDC-UI terminologies for new Edit Asset tab

### Create Data Offer MDS
image

### Create Data Offer non-MDS
image

### Edit Data Offer MDS
image

### Edit Data Offer non-MDS
image

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [ ] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: src/app/routes/connector-ui/asset-page/asset-page/asset-page.component.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export class AssetPageComponent implements OnInit, OnDestroy {
 
   onAssetClick(asset: UiAssetMapped) {
     const onAssetEditClick: OnAssetEditClickFn = (asset, onAssetUpdated) => {
-      this.onEdit(asset, onAssetUpdated);
+      this.router.navigate(['/edit-asset', asset.assetId]);
     };
 
     const buildDialogData = (asset: UiAssetMapped) =>
```
Comment (kulgg) [inline src/app/routes/connector-ui/asset-page/asset-page/asset-page.component.ts:66]: not needed I think?

File: src/app/component-library/edit-asset-form/edit-asset-form/edit-asset-form.component.ts [hunk 2]
```diff
@@ -18,5 +19,10 @@ export class EditAssetFormComponent {
     public form: EditAssetForm,
     public validationMessages: ValidationMessages,
     public expressionFormHandler: ExpressionFormHandler,
+    public activeFeatureSet: ActiveFeatureSet,
   ) {}
+
+  multipleDataSourceOptionsAvailable(): boolean {
+    return this.form.mode !== 'CREATE' && !this.activeFeatureSet.hasMdsFields();
+  }
```
Comment (kulgg) [inline src/app/component-library/edit-asset-form/edit-asset-form/edit-asset-form.component.ts:27]: This logic is not correct

Now MDS Edit looks like this

image

And MDS create does not have the options

image

it should be this probably?

```this.form.mode === 'EDIT' || this.activeFeatureSet.hasMdsFields();```

File: src/app/component-library/edit-asset-form/edit-asset-form/edit-asset-form.component.html [hunk 3]
```diff
@@ -7,59 +7,39 @@
...
-            I do not have a datasource.
-          </mat-radio-button>
           <mat-radio-button value="Datasource">
-            I want to connect to a datasource.
+            Readily available with a new data source
+          </mat-radio-button>
+          <mat-radio-button
+            *ngIf="this.activeFeatureSet.hasMdsFields()"
+            value="On-Request">
+            On Request
```
Comment (kulgg) [inline src/app/component-library/edit-asset-form/edit-asset-form/edit-asset-form.component.html:30]: you can lift line 10 and 17 up to the parent div

**Review by kulgg — CHANGES_REQUESTED** [review]

File: src/app/component-library/edit-asset-form/edit-asset-form/edit-asset-form.component.ts (hunk 2 above)
Comment (illfixit) [reply src/app/component-library/edit-asset-form/edit-asset-form/edit-asset-form.component.ts:27]: it works, thanks

**Review by kulgg — DISMISSED** [review]
LGTM

**Review by kulgg — APPROVED** [review]
LGTM

**Pull request merged**